src/toolbox/setup/fontbm.ts

```diff
@@ -20,15 +21,35 @@ export default async function (): Promise<void> {
 
   // 1. install cmake
   if (system.which('cmake') === null) {
-    spinner.start('Cmake required, installing with Homebrew')
-    await system.exec('brew install cmake')
-    spinner.succeed()
+    if (OS == 'darwin') {
+      spinner.start('Cmake required, installing with Homebrew')
+   	  await system.exec('brew install cmake')
+      spinner.succeed()
+	}
+	else if (OS == 'linux') {
+      spinner.start('CMake required, installing with apt')
+      await execWithSudo(
+        'apt-get install --yes cmake',
```

Comment:
When testing this out in a fresh Linux VM, I had trouble during the `cmake .` step due to missing compiler flags. The fix was to ensure `build-essential` was installed before running `cmake`. While this is probably in most Linux devs' environments already, we can help those few that don't have it yet. 

```suggestion
        'apt-get install --yes build-essential cmake',
```